### ISSUE TITLE
Debug shows detailed/trimesh geometry instead of hull colliders

### ISSUE THREAD

**Issue by Glavin001** [pmndrs/react-three-rapier#69]
# Example Code

```typescript
<Physics>
  <Debug />
  <RigidBody
    colliders={"hull"}
    position={[0, 0, 0]}
  >
    <MiteModel />
  </RigidBody>
</Physics>
```

## Output

See a close fitting detailed/trimesh geometry:

<img width="766" alt="image" src="https://user-images.githubusercontent.com/1885333/188069418-5aa534f7-d2d9-4f0a-99e9-d1781635e889.png">

## Expected

- I would expect to see a convex hull wrapping the outside of the shape (see below)
- When other bodies collide with this rigid hull collider body it acts like a hull, not a detailed/trimesh shape.

![image](https://user-images.githubusercontent.com/1885333/188069964-241dff4f-781e-4e6d-a923-7cdf402b2bb8.png)

---

When I have more time I'll whip up a Codesandbox demo.

**Comment by loukylor** [comment]
Debug also shows `CapsuleColliders` as just cubes ([codesandbox](https://codesandbox.io/s/capsule-colliders-showing-wrong-fq79lz))

**Comment by wiledal** [comment]
The debug collider shapes are indeed not accurate in many cases. I'm working on making this better right now. Including adding more shapes such as RoundedCube, Capsule, more accurate Cylinder etc.

The hull shape is generated within Rapier and I'm using the vertices returned from the collider shape to represent the debug shape, so it's unclear why the vertices would does not represent the hull.

**Comment by wiledal** [comment]
> Debug also shows `CapsuleColliders` as just cubes ([codesandbox](https://codesandbox.io/s/capsule-colliders-showing-wrong-fq79lz)), but like the hull collider, the collider still collides as expected.

This will be fixed in #72

**Comment by wiledal** [comment]
The debug shapes should now be improved in `0.6.7`!

Still not sure about the Hull shape however. As far as I know there's no way to get the exact mesh from the generated Hull used by `rapier`.
